src/ServiceStack.Text/PclExport.NetStandard.cs

```diff
@@ -83,6 +84,15 @@ public NetStandardPclExport()
 #else 
             this.DirSep = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? '\\' : '/';
 #endif
+            var req = HttpWebRequest.Create("http://servicestack.net");
+            try
+            {
+                req.Headers[HttpRequestHeader.UserAgent] = "ServiceStack";
```

Comment:
Ah, long github refreshing, did not see the second comment before starting an answer. 
Yes, this should work, the hardest thing that client should know which properties have to be preinitialized in this way.